### PR TITLE
HTML nesting fixed as main element had no content

### PR DESCRIPTION
**Pull request by alexbowen**
i noticed that the main HTML element didnt actually have any content in it as the nesting of the template was incorrect. this had a few repurcussions in SCSS that i have fixed

Screenshot 2021-02-25 at 12 24 20

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2968.london.cloudapps.digital>

File: app/frontend/src/styles/application/jobseekers/jobseeker.scss [hunk 1]
```diff
@@ -1,4 +1,13 @@
 .jobseeker {
+  &.jobseekers_sessions_new {
+    .govuk-main-wrapper {
+      > .govuk-width-container {
+        margin-top: govuk-spacing(4);
+      }
+    }
+  }
+
+  &.publishers_identifications_new,
```
Comment (alexbowen) [inline app/frontend/src/styles/application/jobseekers/jobseeker.scss:10]: this should really be here but the location/naming of controllers/templates look to need a look at. `jobseeker publishers_identifications_new` are the classes currently applied to the school sign in page as opposed to `jobseeker jobseekers_sessions_new` for teacher sign in page

File: app/views/jobseekers/sessions/new.html.slim [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    h1.govuk-heading-xl = t(".title")
+    h1.govuk-heading-l = t(".title")
```
Comment (david-mears-dfe) [inline app/views/jobseekers/sessions/new.html.slim:5]: Sad

Comment (david-mears-dfe) [reply app/views/jobseekers/sessions/new.html.slim:5]: But I trust your judgement

**Review by david-mears-dfe — APPROVED** [review]

File: app/views/jobseekers/sessions/new.html.slim (hunk 2 above)
Comment (alexbowen) [reply app/views/jobseekers/sessions/new.html.slim:5]: just for consitency with the two sign in pages

Comment (alexbowen) [reply app/views/jobseekers/sessions/new.html.slim:5]: Screenshot 2021-02-25 at 13 16 40
Screenshot 2021-02-25 at 13 16 30

**Pull request merged**